app/slashcommands-open/client/client.ts

```diff
@@ -0,0 +1,55 @@
+import { Meteor } from 'meteor/meteor';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+
+import { roomTypes } from '../../utils/client';
+import { slashCommands } from '../../utils/lib/slashCommand';
+import { Subscriptions, ChatSubscription } from '../../models/client';
+
+function Open(_command: 'open', params: string, _item: Record<string, string>): void {
+	const dict: Record<string, string[]> = {
+		'#': ['c', 'p'],
+		'@': ['d'],
+	};
+
+	let room = params.trim();
+	const type = dict[room[0]];
```

Comment:
since you removed the check for type existing it'd be better to
`const type = dict[room[0]] || {};`